docs/datamodel/links.rst

```diff
@@ -134,6 +139,36 @@ Parameters:
 :eschema:synopsis:`<constraint-declarations>`
     :ref:`Constraint <ref_datamodel_constraints>` declarations.
 
+.. _ref_datamodel_links_on_delete:
+
+On target delete options cover the situation when the target object of
+a link is deleted without explicitly updating the link:
+
+:eschema:synopsis:`restrict`
+    Prohibit deleting the link target as long as the parent object exists.
+    This is the default behavior.
+
+:eschema:synopsis:`delete source`
+    Delete the parent object if any link target is deleted. This may
```

Comment:
I'm not sure that child/parent treminology is the correct one to use here. @elprans?

Reply:
Yeah, let’s stick to calling it *source*.